### ISSUE TITLE
Clickable urls on Log 

### ISSUE THREAD

**Issue by federico9292** [social-manager-tools/social-manager-tools#42]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.6.0
- **Operating System:** Windows 10

### Expected Behavior

Addresses in Log should be links that can be opened in browser.

### Actual Behavior

Addresses are plain text.

### Steps to Reproduce

1. open SocialManagerTools
2. open InstragramBot
3. start
4. open Log

**Comment by ingeniumdesign** [comment]
Request: Delete Log after xx Days

**Comment by ptkdev** [comment]
@ingeniumdesign if you reastart bot, or re-start mode bot delete previously log. 30 days of bot is less than 15mb.

**Comment by ingeniumdesign** [comment]
Yes but can you build a Auto delete? I dont restart all The time 😉
